notifier/notifier.go

```diff
@@ -620,12 +620,12 @@ func (n *Manager) sendAll(alerts ...*Alert) bool {
 			go func(ctx context.Context, client *http.Client, url string, payload []byte, count int) {
 				if err := n.sendOne(ctx, client, url, payload); err != nil {
 					n.logger.Error("Error sending alert", "alertmanager", url, "count", count, "err", err)
```

Comment:
Could you also fix this log message, so it reflects there's a batch of alerts being sent?
```suggestion
					n.logger.Error("Error sending alerts", "alertmanager", url, "count", count, "err", err)
```